test/api/test-draw.c

```diff
@@ -1105,7 +1088,7 @@ test_hb_draw_funcs (const char* font_funcs_name)
   {
     hb_face_t *face = hb_test_open_font_file ("fonts/glyphs.ttf");
     hb_font_t *font = hb_font_create (face);
-    set_font_funcs (font, font_funcs_name);
+    hb_font_set_funcs_using (font, font_funcs_name);
```

Comment:
It probably should assert here (and the two other places below) that `hb_font_set_funcs_using()` returned true.